forum/communications/helpers.py

```diff
@@ -0,0 +1,37 @@
+from rest_framework.serializers import ValidationError
+from .mongo_models import Room
+from startups.models import Startup
+from investors.models import Investor
+
+
+def generate_room_name(participants: list) -> str:
+    room_name = ''
+
+    for participant in participants:
+        room_name += f"{participant['namespace']}_{participant['namespace_id']}"
+
+    room_exists = Room.objects.filter(name=room_name).first()
+    if room_exists:
+        raise ValidationError("Such room already exists.")
+    
+    return room_name
+
+
+def is_namespace_info_correct(namespace_info: dict):
+    user_id = namespace_info.get("user_id")
+    namespace = namespace_info.get("namespace")
+    namespace_id = namespace_info.get("namespace_id")
```

Comment:
I decided with some of the reviewers that there is no need for additional checks because `get()` is used here and if there is no such field, then get() will return None and accordingly the check `if not None:` will pass and an error will be raised.